src/Uno.UWP/Phone/Devices/Notification/VibrationDevice.wasm.cs

```diff
@@ -0,0 +1,44 @@
+﻿#if __WASM__
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Windows.Phone.Devices.Notification
+{
+	public partial class VibrationDevice
+	{
+		private const string JsType = "Windows.Phone.Devices.Notification.VibrationDevice";
+		private static VibrationDevice _instance = null;
+
+		private VibrationDevice()
+		{
+		}
+
+		public static VibrationDevice GetDefault()
+		{
+			if (_instance == null)
+			{
+				var command = $"{JsType}.initialize()";
+				var initialized = Uno.Foundation.WebAssemblyRuntime.InvokeJS(command);
+				if (initialized.Equals("true", StringComparison.InvariantCultureIgnoreCase))
+				{
+					 _instance = new VibrationDevice();
+				}
```

Comment:
You may find a way to cache the result, because it won't start working in the future ;-).